README.md

```diff
@@ -29,7 +29,7 @@ Creates the following resources:
 
 ## Terraform Versions
 
-Terraform 0.12. Pin module version to ~> 2.0.0. Submit pull-requests to master branch.
+Terraform 0.12. Pin module version to ~> 2.0.1. Submit pull-requests to master branch.
```

Comment:
Should we just pin it to 2.X?

Reply:
Yeah, maybe we should just make these `2.X`. I'll update.